src/components/asset-list/EmptyAssetList.js

```diff
@@ -36,11 +37,18 @@ const EmptyAssetList = ({
   const { refresh, isRefreshing } = useRefreshAccountData();
 
   return (
-    <ScrollView
-      contentContainerStyle={{ height: '100%' }}
-      refreshControl={
-        <RefreshControl onRefresh={refresh} refreshing={isRefreshing} />
-      }
+    <ConditionalWrap
+      condition={isWalletEthZero}
```

Comment:
We only want to render a `ScrollView` when the AddFundsInterstitial component is visible (ie. when `isWalletEthZero` is truthy)